python/paddle/fluid/executor.py

```diff
@@ -37,6 +44,19 @@ def switch_scope(scope):
 
 @contextlib.contextmanager
 def scope_guard(scope):
+    """
+    Change the global/default scope instance by Python `with` statement. All
+    variable in runtime will assigned to the new scope.
+
+    Examples:
```

Comment:
Should we use 
```
.. code-block:: python
```
for examples?

Reply:
Only `>>>` style is used in the examples of [napoleon](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).